packages/firestore/package.json

```diff
@@ -79,6 +79,7 @@
   },
   "devDependencies": {
     "@firebase/app": "0.6.12",
+    "@firebase/app-compat": "0.x",
```

Comment:
@Feiyang1 Can you confirm that `app-compat` is fine as a test-dependency? Without it, the browser tests fail to build.